screens/AddUserToChatScreen.js

```diff
@@ -101,9 +101,15 @@ class AddUserToChatScreen extends React.Component {
     };
 
     componentDidUpdate(prevProps) {
-        // if adding user successful, navigate back to chat
+        // if adding user successful, display toast and navigate back to chat
         if (prevProps.addedUser !== this.props.addedUser) {
-            this.props.navigation.navigate('Chat');
+            Toast.show({
+                text: `${this.props.addedUser.displayName} added to chat`,
+                buttonText: "Okay",
+                type: "success",
+                duration: 1000,
```

Comment:
this is just personal preference but i'd make this a higher number because it's hard to read in 1 second

Reply:
I'll prob make it 2 seconds 'cause when I tried out 3 it felt way too long. Lmk if you think differently!